tests/Temporalio.Tests/AssertMore.cs

```diff
@@ -119,6 +119,7 @@ public static void Every<T>(IEnumerable<T> items, params Action<T>[] actions)
             foreach (var item in items)
             {
                 var found = false;
+                var exceptions = new List<Exception>();
```

Comment:
Technically no need for this collection anymore, but no big deal

Reply:
Ah, derp